src/eko/evolution_operator/__init__.py

```diff
@@ -291,7 +286,7 @@ def grid_size(self):
         """Return the grid size."""
         return self.int_disp.xgrid.size
 
-    def mur2_shift(self, q2):
+    def scheme_exponentiated_shift(self, q2):
```

Comment:
Since this function is in a file not dealing with scale variations, which "scheme" you are referring to is a bit ambiguous.

I would suggest a bit more explicit alternative:
```suggestion
    def sv_exponentiated_shift(self, q2):
```

The other option is to put this function in the `sv` module, instead of using `self` you can just have:
```py
def scheme_exponentiated_shift(mu2, xif2, mode):
```
(a good chance to rename this `q2` to `mu2`, and once in the `sv` module there is no ambiguity any longer about which `mode` you are referring to)

Of course also:
```py
def exponentiated_shift(mu2, xif2, mode):
```
works perfectly: it is crystal clear what `exponentiated` means in this context :)